doc/developer/developing.rst

```diff
@@ -286,3 +286,49 @@ Once you have written your own syntax checker, why not `submit a pull request
 <https://github.com/flycheck/flycheck/pulls>`__ to integrate it into Flycheck?
 If it's useful to you, it may be useful for someone else!  Please do check out
 our :ref:`flycheck-contributors-guide` to learn how we deal with pull requests.
+
+Issues with auto-quoting in `flycheck-define-checker`
+-----------------------------------------------------
+
+You may have noticed that lists passed to the ``:command`` or
+``:error-patterns`` in the snippets above are not quoted.  That is because
+`flycheck-define-checker` is a macro which automatically quotes these arguments
+(not unlike `use-package`).
+
+While this makes for a less noisy syntax, it unfortunately prevents you from
```

Comment:
```suggestion
While this makes for less noisy syntax, it unfortunately prevents you from
```

Not sure about this one ^^

Reply:
I'm going to trust the one who is surrounded by native speakers ;)